src/module.js

```diff
@@ -0,0 +1,22 @@
+export function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+export function AppRerender(element) {
+  document.getElementById('app').textContent = '';
+  document.getElementById('app').appendChild(element);
+}
```

Comment:
```suggestion
export function appRerender(element) {
  document.getElementById('app').textContent = '';
  document.getElementById('app').appendChild(element);
}
```
자바스크립트에서 파스칼 케이스는 생성자(클래스)나, jsx 문법을 생각하기가 쉽습니다!!

Reply:
@KJoobin 반영 했습니다~!